internal/resources/metal/vlan/resource.go

```diff
@@ -9,9 +9,10 @@ import (
 	equinix_errors "github.com/equinix/terraform-provider-equinix/internal/errors"
 	"github.com/equinix/terraform-provider-equinix/internal/framework"
 
+	"github.com/equinix/equinix-sdk-go/services/metalv1"
+	packngo "github.com/equinix/equinix-sdk-go/services/metalv1"
```

Comment:
I don't see many references to "packngo" below that would justify keeping the alias. Also - metalv1 is imported twice because of the alias.